core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -302,7 +302,7 @@ class Ruleset {
         }
 
         for (building in buildings.values) {
-            if (building.requiredTech == null && building.cost == 0)
+            if (building.requiredTech == null && building.cost == 0 && !building.uniques.contains("Unbuildable"))
                 lines += "${building.name} must either have an explicit cost or reference an existing tech!"
```

Comment:
This is in mod checker, right? No reason not to update the message, it's untranslated anyway if I remember correctly.